webdriver/tests/bidi/browsing_context/__init__.py

```diff
@@ -37,3 +32,13 @@ def assert_browsing_context(
     assert "url" in info
     assert isinstance(info["url"], str)
     assert info["url"] == url
+
+
+def assert_navigation_info(event, context, url):
```

Comment:
This helper also needs to check the `navigation` which is `null` or a string. Not sure if chromedriver already supports it, but I think that we should have at least a placeholder here, or check for one or the other.

Reply:
So I've tried to cover both cases. Let me know what you think.

Reply:
I think you should also add an argument for navigation given that a call to `browsingContext.navigate` will return this info which you then want to check for the emitted events.

Currently Firefox would fail these checks. I would leave it to Chrome folks to decide what to do for now (and if it's implemented there).